src/test/java/io/github/pmckeown/dependencytrack/TestResourceConstants.java

```diff
@@ -9,6 +9,8 @@ private TestResourceConstants() {
     public static final String V1_METRICS_PROJECT_CURRENT = "/api/v1/metrics/project/(.*)/current";
     public static final String V1_METRICS_PROJECT_REFRESH = "/api/v1/metrics/project/(.*)/refresh";
     public static final String V1_PROJECT_UUID = "/api/v1/project/(.*)";
+    public static final String V1_PROJECT_WITH_PAGINATION = "/api/v1/project";
```

Comment:
The prod code no longer makes a call without the pagination parameters so why would the test code need to?

Reply:
In the test I wanted to express that if we call the GET projects api with just "/api/v1/project" we expect a different set of projects returned. 
In saying that I'm comfortable in dropping this and the associated stub, as the test will still fail if someone changes back to using "/api/v1/project".